api/src/main/java/org/openmrs/hl7/handler/ORUR01Handler.java

```diff
@@ -181,9 +181,6 @@ public Message processMessage(Message message) throws ApplicationException {
 	private Message processORU_R01(ORU_R01 oru) throws HL7Exception {
 		
 		// TODO: ideally, we would branch or alter our behavior based on the
-		// sending application.
-		// String sendingApplication = getSendingApplication(oru);
-		
 		// validate message
```

Comment:
@Nuwantha remove the entire TODO. plus the 'validate message'. do you see how this just repeats the method call?